tests/periph_timer_periodic/tests/01-run.py

```diff
@@ -16,9 +16,9 @@ def testfunc(child):
     start = time.time()
     child.expect_exact('TEST SUCCEEDED')
     end = time.time()
-    # test should run 10 cycles with 100ms each
-    assert (end - start) > 1
-    assert (end - start) < 1.5
+    # test should run 10 cycles with 25ms each
+    assert (end - start) > 0.25
+    assert (end - start) < 0.35
```

Comment:
On atmega (`arduino-uno` and `arduino-mega2560`)  it takes 0.3523.... can we simply increase the margin to fix the test or might something else be going on? Found while running https://github.com/RIOT-OS/Release-Specs/issues/192.

Reply:
Yea pushing it to 40 ms won't hurt much if it helps with slow UARTs.